### PR TITLE
Corrected typographic errors

### PR DESCRIPTION
**Pull request by beremonsivais**
Corrected typographic errors: exclamation marks (¡), accented words, misplaced commas. The changes make the document more presentable. No changes were made to the translation.

- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

**Comment by nthhanh526** [comment]
Xoá phím tắt

**Review by jmeridth — APPROVED** [review]

File: _articles/es/starting-a-project.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ related:
 
 ## El c&oacute;mo y el por qu&eacute; del C&oacute;digo Abierto
 
-&iquest;Est&aacute;s pensando c&oacute;mo comenzar un proyecto de c&oacut;digo abierto? &iexcl;Felicitaciones! El mundo aprecia tu contribuci&oacute;n. Hablemos sobre lo que es un proyecto de c&oacute;digo abierto y porqu&eacute; la gente lo lleva adelante
```
Comment (nthhanh526) [inline _articles/es/starting-a-project.md:15]: Insert:

Comment (nthhanh526) [reply _articles/es/starting-a-project.md:15]: Whitdraw

Comment (nthhanh526) [reply _articles/es/starting-a-project.md:15]: Wall

**Pull request merged**